libs/ui-watt/src/lib/components/input/+storybook/storybook-input-wrapper.component.ts

```diff
@@ -57,7 +57,7 @@ import { FormControl } from '@angular/forms';
     </watt-error>
     <watt-hint *ngIf="hasHint">Some hint</watt-hint>
     <watt-hint *ngIf="hasHint" align="end"
-      >{{ exampleFormControl.value.length }} / 256</watt-hint
+      >{{ exampleFormControl.value?.length }} / 256</watt-hint
```

Comment:
What do we want to leave the user with here if this value is null? :)

Reply:
This is a showcase in Storybook and we know for sure that the `value` will not be `null`/`undefined`.

Reply:
Then why the optional chaining? 😄 

(This is a very minor thing. Just wondering)

Reply:
Reactive forms in Angular v14 are strongly typed. So a `FormControl` used to be untyped but now it is and has a type `string | null`. If we want the previous behaviour, we'll have to use `new UntypedFormControl()`. I didn't do it here because it's Storybook. But I can also make it use the old behaviour and we can migrate to a typed `FormControl` at a later point.

Reply:
Fair enough :) Then lets resolve it here and leave it be. Was just wondering 😄